python/pylibcudf/pylibcudf/tests/test_transform.py

```diff
@@ -81,3 +83,33 @@ def test_one_hot_encode():
         schema=pa.schema([pa.field("", pa.bool_(), nullable=False)] * 3),
     )
     assert result.equals(expected)
+
+
+def test_transform_udf():
+    @cuda.jit(device=True)
+    def op(a, b, c):
+        return (a + b) * c
+
+    ptx, _ = cuda.compile_ptx_for_current_device(
+        op, (numba.float64, numba.float64, numba.float64), device=True
+    )
```

Comment:
Can we just include the PTX (as a string) instead of using numba like in the C++ tests?
```suggestion
    ptx, _ = cuda.compile_ptx_for_current_device(
        op, (numba.float64, numba.float64, numba.float64), device=True
    )
```

Reply:
We could but its [overkill](https://github.com/rapidsai/cudf/pull/17881#discussion_r1945679312). 

Aside: Its awesome that `cuda.compile_ptx_for_current_device` exists in python.